proposals/vip_6.ts

```diff
@@ -0,0 +1,39 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const vip_6: ProposalDescription = {
+  title: 'VIP-6: PSM Restructuring',
+  commands: [
+    {
+      target: 'feiPriceBoundPSM',
+      values: '0',
+      method: 'pauseMint()',
+      arguments: [],
+      description: 'Pause Minting on the FEI PSM'
+    },
+    {
+      target: 'pcvGuardian',
+      values: '0',
+      method: 'withdrawAllERC20ToSafeAddress(address, address)',
+      arguments: ['{feiPriceBoundPSM}', '{volt}'],
+      description: 'Remove all VOLT from FEI PSM'
+    },
+    {
+      target: 'pcvGuardian',
+      values: '0',
+      method: 'addWhitelistAddress(address)',
+      arguments: ['{daiPriceBoundPSM}'], // todo add to list of addresses
+      description: 'Add DAI PSM to whitelisted addresses on PCV Guardian'
+    },
+    {
+      target: 'usdcPriceBoundPSM',
+      values: '0',
+      method: 'unpauseRedeem()',
+      arguments: [],
+      description: 'Unpause redemptions for USDC PSM'
+    }
```

Comment:
Shouldn't this proposal also send VOLT into the new DAI PSM?

Reply:
Ideally, but we can't pass a variable amount in the current proposal structure, so we'd have to withdraw it away from the FEI psm and then in another proposal deposit it into the DAI PSM so we know the exact amount